api/camunda/camunda-openapi.yaml

```diff
@@ -1537,9 +1381,16 @@ paths:
               schema:
                 $ref: "#/components/schemas/CreateProcessInstanceResult"
         "400":
-          description: The provided data is not valid.
+          $ref: "#/components/responses/InvalidData"
         "500":
           $ref: "#/components/responses/InternalServerError"
+        "504":
+          description: |
+            The process instance creation request timed out in the gateway.
+
+            This can happen if the `awaitCompletion` request parameter is set to `true`
+            and the created process instance did not complete within the defined request timeout.
+            This often happens when the created instance is not fully automated or contains wait states.
```

Comment:
This is missing `content`. Will be added upstream with https://github.com/camunda/camunda/pull/28817